util/common.js

```diff
@@ -0,0 +1,21 @@
+const { flatten } = require("flat");
+
+class CommonUtils {
```

Comment:
This is interesting, why would we want to expose the utilities as static functions of a class defined in this file. Rather than expose them as functions itself ?

Reply:
we will be moving to typescript soon so this is inline with typescript style

Reply:
I want to understand the basis of it. I mean why would that sort of thing is thought to be good over exposing the individual functions ?

Reply:
Easy to mock if required using jest.